docs/release-notes.md

```diff
@@ -32,6 +32,7 @@
 * [ZNTA-905](https://zanata.atlassian.net/browse/ZNTA-905) - Remove 0% matching translation memory entry
 * [ZNTA-928](https://zanata.atlassian.net/browse/ZNTA-928) - Readonly project doesn't have "lock" icon in UI
 * [ZNTA-793](https://zanata.atlassian.net/browse/ZNTA-793) - Remove unused method in TranslationMemoryAction
+* [ZNTA-1017](https://zanata.atlassian.net/browse/ZNTA-1017) - Refactor translation update event 
```

Comment:
I don't think refactoring tasks like ZNTA-1017 or ZNTA-793 should appear in the release notes.